spring/spring/pom.xml

```diff
@@ -237,6 +237,43 @@
                     </instructions>
                 </configuration>
             </plugin>
+            <plugin>
```

Comment:
Currently we support both Spring 3 (placed in this module) and Spring 4 (placed here https://github.com/infinispan/infinispan/tree/master/spring/spring4). Please add similar changes to Spring 4. Spring 3 support will be dropped soon.